python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -229,6 +229,10 @@ def preprocess_ondisk_dataset(
             in_memory = (
                 True if "in_memory" not in feature else feature["in_memory"]
             )
+            if feature["domain"] == "edge" and not include_original_edge_id:
```

Comment:
nit: this will raise multiple warnings.
What about add has_edge_feature_data in the for loop, and raise one warning after the for loop?